src/applications/claims-status/utils/helpers.js

```diff
@@ -52,6 +65,22 @@ export function getUserPhase(phase) {
   return phase - 3;
 }
 
+function isInEvidenceGathering(claim) {
+  const allowedClaimTypes = ['evss_claims', 'claim'];
+  const isEvssClaim = claim.type.toLowerCase() === 'evss_claims';
```

Comment:
I don't think we need to worry about lowercasing these, they should always be either either `evss_claims` or `claim`. For EVSS for example, the type field is set by vets-api